src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1608,11 +1608,8 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserInSectionBy
                     frc.commentGiver = Const.DISPLAYED_NAME_FOR_ANONYMOUS_PARTICIPANT;
                 }
 
-                if (responseComments.get(frc.feedbackResponseId) == null) {
-                    responseComments.put(frc.feedbackResponseId,
-                            new ArrayList<FeedbackResponseCommentAttributes>());
-                }
-                responseComments.get(frc.feedbackResponseId).add(frc);
+                responseComments.computeIfAbsent(frc.feedbackResponseId, String -> new ArrayList<>())
```

Comment:
I'm surprised CheckStyle didn't flag this. `String` here is a lambda parameter and so it needs to follow camelCase, e.g. it should be `string`. Or if you want to be constant with you've done in another file, use `key`.

Reply:
I will use key instead to maintain consistency.